rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/helper/FrescoHelper.kt

```diff
@@ -0,0 +1,43 @@
+package chat.rocket.android.widget.helper
+
+import android.net.Uri
+import android.support.graphics.drawable.VectorDrawableCompat
+import chat.rocket.android.widget.R
+import com.facebook.drawee.backends.pipeline.Fresco
+import com.facebook.drawee.drawable.ProgressBarDrawable
+import com.facebook.drawee.generic.GenericDraweeHierarchy
+import com.facebook.drawee.view.SimpleDraweeView
+
+class FrescoHelper {
+    companion object {
+        @JvmStatic fun loadImage(draweeView: SimpleDraweeView, imageUri: String) {
```

Comment:
Why not an extension function to `SimpleDraweeView` instead of static methods?

```
fun SimpleDraweeView.loadImage(uri: String) {
    imageUri = uri
}
```